### PR TITLE
Improve install generator and tweak views

### PR DESCRIPTION
**Pull request by anarchivist**

### CODE REVIEW

File: app/views/krikri/harvest_sources/index.html.erb [hunk 1]
```diff
@@ -21,5 +26,6 @@
 
 <%= link_to 'New',
     new_institution_harvest_source_path(institution_id: @institution.id)
-%>
+%> |
```
Comment (no-reply) [inline app/views/krikri/harvest_sources/index.html.erb]: Is this pipe just to separate links?  Would this be better in a stylesheet?

Comment (anarchivist) [inline app/views/krikri/harvest_sources/index.html.erb]: Yes. And probably. :) I can consult with @AudreyAltman and push up a fix. (Alternately I can break out this out into a separate PR.)

Comment (AudreyAltman) [inline app/views/krikri/harvest_sources/index.html.erb]: I think we had decided to wait on all non-urgent styling. We can fix it now if you would like, but I would be okay with leaving it until we start styling work in earnest.

Comment (anarchivist) [inline app/views/krikri/harvest_sources/index.html.erb]: @AudreyAltman :+1: OK, no sweat. I'll close this and resubmit the PR with just one commit for the generator.

Comment (no-reply) [inline app/views/krikri/harvest_sources/index.html.erb]: :cool: :sunglasses:

**Pull request closed without merging**